x/stake/keeper/validator_test.go

```diff
@@ -112,7 +112,7 @@ func TestUpdateBondedValidatorsDecreaseCliff(t *testing.T) {
 		val := types.NewValidator(Addrs[i], PKs[i], types.Description{Moniker: moniker})
 		val.BondHeight = int64(i)
 		val.BondIntraTxCounter = int16(i)
-		val, pool, _ = val.AddTokensFromDel(pool, int64((i+1)*10))
+		val, pool, _ = val.AddTokensFromDel(pool, sdk.NewInt(int64((i+1)*10)))
```

Comment:
We should be able to remove int64 cast here right?

Reply:
The data type of variable i is int. This cast can't be removed.

Reply:
Oh I see, newInt takes int64